ipython/ResamplingAndMonteCarloMethods/resampling_tutorial_2c.ipynb

```diff
@@ -35,7 +35,7 @@
    "id": "341c6722-3240-4baa-854f-6cab6d397557",
    "metadata": {},
    "source": [
-    "The text shows the results of analysis using Spearman's correlation coeefficient, a statistic sensitive to linear association between the ranks of the samples. Specifically, the null hypothesis that there is no association between total collagen and free protein is tested against the alternative that there is a positive, linear association between the two."
+    "The text shows the results of analysis using Spearman's correlation coefficient, a statistic sensitive to linear association between the ranks of observations within the two samples. Specifically, the null hypothesis that there is no association between total collagen and free protein is tested against the alternative that there is a positive association between the two (whether linear or not)."
    ]
```

Comment:
This might still need some clarification, but the original made it seem like the alternative was that there's a linear association between total collagen and free protein, not just their ranks. It should be pointed out that a linear association in ranks corresponds to any monotonic association at all.